### ISSUE TITLE
StorasBot ignores commands from bot

### ISSUE THREAD

**Issue by ceilingwaffle** [ekgame/storasbot#4]
I created a discord bot named "wafflebot". I gave it "Manage Server" permissions in Discord. But when I use the bot to send "tracking command" messages via Python, StorasBot does not respond, and the players listed in $tracking remains unchanged. Example: http://puu.sh/tXUsb/ef2aeb29bb.png

My main Discord account (non-bot) also has Manage Server permissions, but StorasBot successfully responds to my own commands.

Is StorasBot purposely ignoring commands from bots, or am I doing something wrong?

Thanks

**Comment by ekgame** [comment]
Yes, the commands from bots are ignored by design, mostly to avoid infinite loop of `$bin` commands. If you have a good reason for automating some commands, there may be a better way to solve the issue.
